### ISSUE TITLE
Android Custom Tabbed Page Renderer causes NullReferenceException

### ISSUE THREAD

**Issue by ianvink** [xamarin/Xamarin.Forms#1462]
### Description
Android Custom Tabbed Page Renderer causes NullReferenceException

### Steps to Reproduce

Create a TabbedPage Renderer on XF 2.5 and show a basic tabbed page using it:

         [assembly: ExportRenderer(typeof(RootTab), typeof(CoApp.Droid.Renderers.RootTabRenderer))]

    namespace CoApp.Droid.Renderers
    {
     public class RootTabRenderer : TabbedPageRenderer, TabLayout.IOnTabSelectedListener
     {

        private RootTab _page;

        public RootTabRenderer(Context context) : base(context)
        {
        }

        protected override void OnElementChanged(ElementChangedEventArgs<TabbedPage> e)
        {
            base.OnElementChanged(e);
            if (e != null)
            {
                if (e.NewElement != null)
                {
                    _page = (RootTab) e.NewElement;
                }
                else
                {
                    _page = (RootTab) e.OldElement;
                }
            }

        }

        async void TabLayout.IOnTabSelectedListener.OnTabReselected(TabLayout.Tab tab)
        {
            if (_page?.CurrentPage?.Navigation != null)
                await _page.CurrentPage.Navigation.PopToRootAsync();
        }
        }
    }

From HockeyApp
```

Xamarin caused by: android.runtime.JavaProxyThrowable: System.NullReferenceException: Object reference not set to an instance of an object
Xamarin.Forms.Platform.Android.AppCompat.Platform.Xamarin.Forms.Platform.Android.IPlatformLayout.OnLayout(bool changed, int l, int t, int r, int b)<aa25dff89f734b22b9ab88fd01c78705>:0
Xamarin.Forms.Platform.Android.PlatformRenderer.OnLayout(bool changed, int l, int t, int r, int b)<aa25dff89f734b22b9ab88fd01c78705>:0
Android.Views.ViewGroup.n_OnLayout_ZIIII(IntPtr jnienv, IntPtr native__this, bool changed, int l, int t, int r, int b)<28e323a707a2414f8b493f6d4bb27c8d>:0
at (wrapper dynamic-method) System.Object:a5e32371-d898-4fc4-abf9-46e453ff61e8 (intptr,intptr,bool,int,int,int,int)
md5b60ffeb829f638581ab2bb9b1a7f4f3f.PlatformRenderer.n_onLayout(Native Method)
md5b60ffeb829f638581ab2bb9b1a7f4f3f.PlatformRenderer.onLayout()PlatformRenderer.java:63
android.view.View.layout()View.java:17637
android.view.ViewGroup.layout()ViewGroup.java:5575
android.widget.RelativeLayout.onLayout()RelativeLayout.java:1079
android.view.View.layout()View.java:17637
android.view.ViewGroup.layout()ViewGroup.java:5575
android.widget.FrameLayout.layoutChildren()FrameLayout.java:323
android.widget.FrameLayout.onLayout()FrameLayout.java:261
android.view.View.layout()View.java:17637
android.view.ViewGroup.layout()ViewGroup.java:5575:
```

### Basic Information

Xamarin Forms 2.5
Android Emulator
Windows Visual Studio

**Comment by samhouts** [comment]
[Issue1462.zip](https://github.com/xamarin/Xamarin.Forms/files/1599061/Issue1462.zip)

I'm unable to reproduce this with a simple project. Can you please alter the attached project so that it exhibits the issue? Thanks!

**Comment by jassmith** [comment]
Closing due to inactivity

**Comment by gtbX** [comment]
I came across a similar issue implementing TabLayout.IOnTabSelectedListener.OnTabReselected() in a custom TabbedPageRenderer. I was able to work around it by implementing the remaining methods of the IOnTabSelected interface:

		void TabLayout.IOnTabSelectedListener.OnTabSelected(TabLayout.Tab tab) { }

		void TabLayout.IOnTabSelectedListener.OnTabUnselected(TabLayout.Tab tab) { }

though I'm not certain I understand what's going on. The IOnTabSelected implementation in the base class is private so I can't call them directly from the subclass. Yet the TabBar still seems to work switching tabs correctly, even calling my custom OnTabReselected(). 🤷‍♂️
